### DISCUSSION TITLE
Can i'm host figura proxy on my pc?

### DISCUSSION THREAD

**Discussion by timylya777** [FiguraMC/Figura#292] [Q&A]
Hello everyone, i have a question: can i'm host figura proxy on my pc? and if i can - how i can it?

**Comment by PoolloverNathan** [comment]
the figura *backend*? you can, but we're not allowed to say how

**Comment by Riftlight** [comment]
You can't easily at all. If you want a serverside backend that's tied to the server, check out FSB in the discord.
